search-commons/src/test/java/no/unit/nva/search/common/builder/PartOfQueryTest.java

```diff
@@ -0,0 +1,18 @@
+package no.unit.nva.search.common.builder;
+
+import static org.junit.Assert.assertNotNull;
+
+import no.unit.nva.search.resource.ResourceParameter;
+
+import org.junit.jupiter.api.Test;
+
+class PartOfQueryTest {
+
+    @Test
+    void checkPartOfQuery() {
+        var partOfQuery = new PartOfQuery<ResourceParameter>();
+        assertNotNull(
+                partOfQuery.buildMatchAnyValueQuery(
+                        ResourceParameter.CONTRIBUTORS_OF_CHILD, "ewsrdf"));
```

Comment:
What is ewsrdf?

Reply:
This test is only to satisfy 100% codecoverage, buildMatchAnyValueQuery is never used by PartOfQuery, @JacocoGenerated, doesn't help here...